ligo/stablecoin/spec.ligo

```diff
@@ -288,7 +289,7 @@ type owner is address
 type operator is address
 
 type operators is
-  big_map ((owner * operator), unit)
+  big_map ((owner * (operator * token_id)), unit)
```

Comment:
What's the point to store `token_id` here? Isn't it always 0?

Reply:
This construct makes sense for multiasset contracts, but for singleasset (like stablecoin) it looks redundant.